docs/running-in-production/docker-compose.md

```diff
@@ -44,8 +44,8 @@ Read more in the [authentication documentation](./authentication.md).
   which will output multiple columns of information about your running container(s). Look for the `prairielearn/prairielearn` image and copy its corresponding name. For example, the name of the PrairieLearn container in this `docker ps` output is `upbeat_roentgen`:
 
   ```console
-  CONTAINER ID  IMAGE                      COMMAND              CREATED      STATUS      PORTS                   NAMES
-  e0f522f41ea4  prairielearn/prairielearn  "/bin/sh -c /Prai…"  2 hours ago  Up 2 hours  0.0.0.0:3000->3000/tcp  upbeat_roentgen
+  CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES
+  e0f522f41ea4 prairielearn/prairielearn "/bin/sh -c /Prai…" 2 hours ago Up 2 hours 0.0.0.0:3000- upbeat_roentgen > 3000/tcp
```

Comment:
Another one 👀

Reply:
Resolved in [61d9c39](https://github.com/PrairieLearn/PrairieLearn/pull/11641/commits/61d9c3994d046a89928e9d27878becac6a0c46d8).